samcli/lib/providers/sam_stack_provider.py

```diff
@@ -55,14 +55,14 @@ def __init__(
         self._resources = self._template_dict.get("Resources", {})
         self._global_parameter_overrides = global_parameter_overrides
 
-        LOG.debug("%d stacks found in the template", len(self._resources))
-
         # Store a map of stack name to stack information for quick reference -> self._stacks
         # and detect remote stacks -> self._remote_stack_full_paths
         self._stacks: Dict[str, Stack] = {}
         self.remote_stack_full_paths: List[str] = []
         self._extract_stacks()
 
+        LOG.debug("%d nested stacks found in the template", len(self._stacks))
```

Comment:
`self._stacks` contains the root stack as well, I think it is better to leave it `stacks found ...` instead of `nested stacks ...`

Reply:
@moelasmar cool - I've done the amendment